rainier-plot/src/main/scala/com/stripe/rainier/plot/Jupyter.scala

```diff
@@ -143,4 +187,39 @@ object Jupyter {
     baos.close
     array
   }
+
+  //copied from private evilplot class
+  private case class SurfacePlotRenderer(
```

Comment:
I don't see this referenced anywhere. Was it part of an older iteration on this code?